### ISSUE TITLE
[chocolatey] Publication fails with Chocolatey-1.1.0 - description missing

### ISSUE THREAD

**Issue by myron0815** [jreleaser/jreleaser#950]
### Task List

- [X] Steps to reproduce provided
- [X] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [X] Full description of the issue provided (see below)

### Steps to Reproduce

1. package with chocolatey-1.1.0 - mvn jreleaser:package

### Expected Behaviour

Should take global pom ${project.description} value

### Actual Behaviour

nuspec xml is generated with empty description tag.  
${project.description} is set in pom.xml (and also in jreleaser.project.description) 
It seems, choco needs that with 1.1 (first-time user - have no experience if that worked with previous versions)

Tell us what happens instead

```
[INFO]   - Packaging app distribution
[INFO]     [chocolatey] packaging app distribution
[INFO]     [chocolatey] Chocolatey v1.1.0
[INFO]     [chocolatey] Description is required.
[WARNING]     [chocolatey] failure: Unexpected error
```

### Environment Information

- **Operating System**: Windows 10
- **JReleaser Version:** actual 1.2.0
- **JDK Version:** Eclipse Adoptium\jdk-11.0.16.8-hotspot

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

```
[INFO]      [chocolatey] packaging app distribution
[DEBUG]     [chocolatey] creating props for app/chocolatey
[DEBUG]     [chocolatey] filling distribution properties into props
[DEBUG]     [chocolatey] filling git properties into props
[DEBUG]     [chocolatey] filling artifact properties into props
[DEBUG]     [chocolatey] filling packager properties into props
[DEBUG]     [chocolatey] copying files from C:\Users\*****\WORKSPACES\*****\target\jreleaser\prepare\app\chocolatey to C:\Users\*****\WORKSPACES\*****\target\jreleaser\package\app\chocolatey
[DEBUG]     [chocolatey] choco pack app.nuspec
[INFO]      [chocolatey] Chocolatey v1.1.0
[INFO]      [chocolatey] Description is required.
[WARN]      [chocolatey] failure: Unexpected error
org.jreleaser.model.packager.spi.PackagerProcessingException: Unexpected error
	at org.jreleaser.packagers.AbstractPackagerProcessor.executeCommand(AbstractPackagerProcessor.java:229)
	at org.jreleaser.packagers.ChocolateyPackagerProcessor.createChocolateyPackage(ChocolateyPackagerProcessor.java:156)
	at org.jreleaser.packagers.ChocolateyPackagerProcessor.doPackageDistribution(ChocolateyPackagerProcessor.java:76)
	at org.jreleaser.packagers.AbstractTemplatePackagerProcessor.doPackageDistribution(AbstractTemplatePackagerProcessor.java:138)
	at org.jreleaser.packagers.AbstractPackagerProcessor.packageDistribution(AbstractPackagerProcessor.java:161)
	at org.jreleaser.packagers.ChocolateyPackagerProcessor.packageDistribution(ChocolateyPackagerProcessor.java:56)
	at org.jreleaser.engine.distribution.DistributionProcessor.packageDistribution(DistributionProcessor.java:108)
	at org.jreleaser.engine.distribution.Distributions.processPackager(Distributions.java:126)
	at org.jreleaser.engine.distribution.Distributions.processDistribution(Distributions.java:103)
	at org.jreleaser.engine.distribution.Distributions.process(Distributions.java:89)
	at org.jreleaser.workflow.PackageWorkflowItem.doInvoke(PackageWorkflowItem.java:37)
	at org.jreleaser.workflow.AbstractWorkflowItem.lambda$invoke$0(AbstractWorkflowItem.java:38)
	at org.jreleaser.engine.hooks.CommandHookExecutor.execute(CommandHookExecutor.java:52)
	at org.jreleaser.workflow.AbstractWorkflowItem.invoke(AbstractWorkflowItem.java:38)
	at org.jreleaser.workflow.WorkflowImpl.execute(WorkflowImpl.java:70)
	at org.jreleaser.maven.plugin.JReleaserPackageMojo.execute(JReleaserPackageMojo.java:78)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:370)
	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:351)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:215)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:171)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:163)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
Caused by: org.jreleaser.util.command.CommandException: Command execution error. exitValue = 1
	at org.jreleaser.packagers.AbstractPackagerProcessor.executeCommand(AbstractPackagerProcessor.java:226)
	... 39 more
```

**Comment by aalmiray** [comment]
The first thing I notice that the log does mention that Chocolatey requires a description. However, the message is logged as `INFO` but it should be upgraded to `ERROR` and fail the `prepare` step as early as possible.

**Comment by myron0815** [comment]
ups, sorry, should have pressed F5 to see your changes ;)

**Comment by myron0815** [comment]
Generating chocolatey stubs with `jreleaser:template` showed, that `<description>{{projectLongDescription}}</description>` is used.

If you set this longDescription up, it works nicely.  
But from a Maven POV, this is usually not default.  
We might take the short description as fallback, if you prefer to have the long one in there...

**Comment by aalmiray** [comment]
The value of `project.description` is used as default if `project.longDescription` is not set. If this is not the case then that’s the first thing to fix. 

Do you see values for both properties when running `mvn jreleaser:config`?

**Comment by myron0815** [comment]
Nope.  
Just `project.description` is set (when executing via Maven)

**Comment by aalmiray** [comment]
Released in v1.3.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.3.0
